crates/ruma-html/Cargo.toml

```diff
@@ -19,7 +19,7 @@ unstable-msc4286 = []
 
 [dependencies]
 as_variant = { workspace = true }
-html5ever = "0.29.0"
+html5ever = "0.35.0"
```

Comment:
This would be a breaking change too. We have types from markup5ever in the public API.